pipeline/data_store.py

```diff
@@ -16,15 +16,15 @@
 
 
 UPSTREAM_NAMES = {
-    'exposure': [], # no upstreams
+    'exposure': [],  # no upstreams
     'preprocessing': ['exposure'],
     'extraction': ['preprocessing'],
     'astro_cal': ['extraction'],
     'photo_cal': ['extraction', 'astro_cal'],
     'subtraction': ['reference', 'preprocessing', 'extraction', 'astro_cal', 'photo_cal'],
     'detection': ['subtraction'],
     'cutting': ['detection'],
-    'measurement': ['detection', 'photo_cal'],
+    'measuring': ['cutting'],
```

Comment:
This isn't important, but... we're mixing different kinds of nouns here (gerunds and non-gerunds).  We have subtraction instead of subtracting, but then measuring instead of measurement.

Reply:
Yeah I know it is annoying. But I don't know if extraction->extracting or detection->detecting is a real improvement. 

I am sure that changing measurement (the verb) to differentiate it from measurements (the noun) is a step in the right direction.